bazel/repositories.bzl

```diff
@@ -241,6 +253,42 @@ def envoy_dependencies(path = "@envoy_deps//", skip_com_google_protobuf = False,
     go_deps(skip_targets)
     envoy_api_deps(skip_targets)
 
+def com_github_bombela_backward(repository = ""):
+  native.new_git_repository(
+      name = "com_github_bombela_backward",
+      remote = "https://github.com/bombela/backward-cpp",
```

Comment:
Thoughts on reworking https://github.com/envoyproxy/envoy/blob/master/bazel/repository_locations.bzl so that it contains both locations and versions for all deps? IMO it's a lot easier to look in that file to grok what is being pulled and at what SHA. Then we can just reference that in this file like we already kind of do for some things.

Reply:
I'm willing, but would like to do that as a separate PR to avoid diff noise.

Reply:
OK thank you. Sounds great in follow up PR.